backend/jest.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  testEnvironment: 'node',
+  transform: {
+    '^.+\\.tsx?$': 'ts-jest',
+  },
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
+  testRegex: '(/__tests__/.*|(\\.|/)(test|spec))\\.(ts|js)x?$',
```

Comment:
Putting tests like that is bad idea. Tests should be added to every module independently 
because this solution is more scalable. Check frontend tests for reference

Reply:
testRegex removed, added testFileStructureConvention dir in src/ which follows mentioned convention (module/tests) as a pattern for further testing